tests/mlmodel_bedrock/conftest.py

```diff
@@ -33,7 +33,8 @@
     "transaction_tracer.stack_trace_threshold": 0.0,
     "debug.log_data_collector_payloads": True,
     "debug.record_transaction_failure": True,
-    "ml_insights_event.enabled": True,
+    "machine_learning.enabled": True,
```

Comment:
I know you said it was failing without this but I ran the tests locally without this setting and openai doesn't have this set and they both work. There must have been something else going on there.
```suggestion
```